packages/devextreme/testing/tests/DevExpress.ui.widgets/list.markup.tests.js

```diff
@@ -524,13 +527,36 @@ QUnit.module('decorators markup', {}, () => {
             selectAllText: 'Test'
         }));
 
-        const $multipleContainer = $list.find('.dx-list-select-all');
+        const $multipleContainer = $list.find(toSelector(SELECT_ALL_CLASS));
         assert.equal($multipleContainer.length, 1, 'container for SelectAll rendered');
         assert.equal($multipleContainer.text(), 'Test', 'select all rendered');
         const $checkbox = $multipleContainer.find('.dx-checkbox');
         assert.equal($checkbox.length, 1, 'checkbox rendered');
     });
 
+    QUnit.test('selectAll text and aria-label attribute should be equal to custom localized text (T1239880)', function(assert) {
+        localization.loadMessages({ 'en': { 'dxList-selectAll': 'custom-select-all' } });
+        localization.locale('en');
+
+        const $list = $($('#list').dxList({
+            dataSource: [
+                { id: 1, text: 'Item 1' },
+                { id: 2, text: 'Item 2' },
+                { id: 3, text: 'Item 3' },
+            ],
+            showSelectionControls: true,
+            selectionMode: 'all',
+        }));
+
+        const $selectAllCheckBox = $list.find(toSelector(SELECT_ALL_CHECKBOX_CLASS));
+        const $multipleContainer = $list.find(toSelector(SELECT_ALL_CLASS));
+
+        assert.equal($selectAllCheckBox.attr('aria-label'), 'custom-select-all', 'selectAll checkbox aria-label should be equal to localized text');
```

Comment:
Actually no diff in this case, but it's better to not break the habit
```suggestion
        assert.strictEqual($selectAllCheckBox.attr('aria-label'), 'custom-select-all', 'selectAll checkbox aria-label should be equal to localized text');
```

Reply:
[b901b96](https://github.com/DevExpress/DevExtreme/pull/27664/commits/b901b967eb648ba2c37c60a7dcfab427de681ee8)